### PR TITLE
Add a settings button to the first time group header.

### PR DESCRIPTION
**Pull request by roundhill**
Opens the notifications settings when clicked which currently just opens them in a new window since that's about all we can do from the iframe version.

@dmsnell maybe you could add your middleware solution to get it working to open the route directly when in Calypso?

screen shot 2017-05-18 at 4 48 16 pm

Refs #104

### CODE REVIEW

**Comment by dmsnell** [comment]
@roundhill found a bug where if there were no notes for `Today` then the settings icon wouldn't show and then I moved the window opening to middleware.

we're not yet at the coupling point with Calypso so I'm not sure if we can reasonably navigate Calypso to the settings from this PR as that will take some additional work to make happen.

**Comment by dmsnell** [comment]
On second thought @roundhill I went ahead and added in the custom middleware idea. We can refine it but I think it will work. We will have to coordinate the version bump in Calypso with this because at the moment the settings icon won't do anything in Calypso until we create the corresponding custom middleware there.

File: src/state/index.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ const reducer = combineReducers({
...
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
-const withMiddleware = composeEnhancers(applyMiddleware(actionMiddleware))(createStore);
+const withMiddleware = customMiddleware =>
+    composeEnhancers(applyMiddleware(actionMiddleware(customMiddleware)))(createStore);
 
-export const store = withMiddleware(reducer, reducer(undefined, { type: '@@INIT' }));
+export let store = { dispatch: () => {}, getState: () => {} };
+
+export const init = ({ customMiddleware }) =>
+    (store = withMiddleware(customMiddleware)(reducer, reducer(undefined, { type: '@@INIT' })));
```
Comment (dmsnell) [inline src/state/index.js]: @gwwar @coderkevin see above for the most primitive version of my custom integration middleware injection idea

File: src/templates/gridicons.jsx [hunk 2]
```diff
@@ -238,6 +238,20 @@ export default React.createClass({
                         </g>
                     </svg>
                 );
+
+            case 'gridicons-cog':
+                return (
+                    <svg {...sharedProps} xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+                        <title>Arrow Left</title>
```
Comment (drw158) [inline src/templates/gridicons.jsx]: Should be `Cog`

Comment (dmsnell) [inline src/templates/gridicons.jsx]: Should be fixed

**Comment by roundhill** [comment]
Tried it out in Calypso 😍

**Pull request merged**